README.md

```diff
@@ -1,29 +1,17 @@
-![](https://img.shields.io/badge/SDK-v13.0.0-blue) <Please check version is the same as specified in requirements.txt>
+![](https://img.shields.io/badge/SDK-v13.6.2-blue) <Please check version is the same as specified in requirements.txt>
 
-# CPT interpretation with Robertson
-This sample app shows  show how to interpret a CPT (.gef) to a soil layout using the Robertson method. <basic one line intro here>
+# CPT interpretation
+This sample app shows show how to interpret a CPT (.gef) to a soil layout using a classification method of choice. <basic one line intro here>
 
-This app can consist of multiple projects with multiple cpt files. The first picture is an example of such a cpt file. On the left there is the input where the minimum layer thickness can be defined. 
-On the right the soil layer interpretation is shown with its data.
+This app can consist of a single editor with steps to guide the user. The image below is an example of a result of a cpt file interpretation. 
+On the left there is the input where the interpreted results can be adjusted manually. On the right the soil layer interpretation is shown with its data.
 
 ![](resources/cpt_visualisation.png)
```

Comment:
rename this file  (see comment about standardised cover_image)

Reply:
The cover image is, although slightly, different from the cpt_visualization.png image.